WordPress/gutenberg
# List: Add Align Support

## What?

Added Align Wide and full width support to the List block.

Part of : https://github.com/WordPress/gutenberg/issues/43248

## Why?

The block was missing Align Support (Wide and Full Width) Support Features.

## Testing Instructions
- Test the `List block` in both the block editor and site editor.
- Confirm that the wide width, full width work correctly, and display correctly on the front.
- Depending on where you place the block, you may need to place it inside a group with the "inner blocks use content width" setting enabled.
- In the Site Editor, open or create the template for single posts. Add a `List block` compare the position of the content inside these blocks, with different settings enabled. It should be possible to align the block so that the text position is the same in `List Block`.

## Screenshots or Screencast 


https://github.com/user-attachments/assets/7f083361-9b3a-4a5a-911c-c76cf53c1b7c
